content/deploy-troubleshooting.md

```diff
@@ -18,13 +18,13 @@ Note that code-level review lies outside the scope of Galaxy's support. If this
 
 <h2 id="503-errors">503 errors or Deployment Failures</h2>
 
-Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This means no healthy containers are currenty available to serve your app.
+Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This means no healthy containers are currently available to serve your app.
 
 There are several potential reasons for this. One example is that all containers are unhealthy, because they are all stuck in a CPU loop. Another reason is that none are running, because they all recently crashed (especially if the total number of your containers is 1, and it hasn't had time to restart). Another reason is because your build failed, if this is the first time you're deploying a container for that app or if the only other available containers built successfully but are unhealthy.
 
 The most common cause for this error is a problem in your code that prevents deployment.
 
-In some cases you may see the 503 error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command. If you've waited 15 minutes or more, the problem will almost surely not resolve itself by waiting longer.
+In some rare cases you may see the 503 error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command. Waiting should resolve this. If you've waited 15 minutes or more, however, the problem will almost surely not resolve itself by waiting longer.
```

Comment:
I see; I think I've figured out my misunderstanding here. I thought you meant that 503 errors could happen if you checked too quickly, on rare occasions; but I think what you're saying is, there shouldn't be any 503 errors after 1-2 minutes, remove this. I can do that.

The '15 mins or longer' thing comes from users seeing 503 errors and thinking, I'll wait it out; it's really meant to say to them, if you see a 503 error, waiting won't make it go away.

So in summary, it sounds like the whole highlighted green '503 error temporarily' section can be safely removed. If that sounds good to you, let me know and I'll do it.